src/preprocessing/passes/ackermann.h

```diff
@@ -64,6 +66,15 @@ class Ackermann : public PreprocessingPass
   /* Map each function term to the new Skolem variable created by
    * ackermannization */
   theory::SubstitutionMap d_funcToSkolem;
+  /* Map each uninterpreted sort term to the new Skolem variable created by
+   * ackermannization */
+  theory::SubstitutionMap d_sortsToSkolem;
+  /* Map each uninterpreted sort term to a pair of integers.
+   * The first value is the lowest capacity that the targeting BV should have
+   * The second value is the size of the BV to which the sort will actually be
```

Comment:
https://github.com/CVC4/CVC4/pull/3372#discussion_r334044329
Didn't we agree that the first number is enough here? That is, we can always assign bit-vectors of that size, without needing to increasing it for each distinct sort?
If this is the case, we don't need `second` here.

Reply:
I was thinking of including that information in the class, because I don't want to assume the user knows the algorithm of assigning lengths. Do you still think we should remove it?

Reply:
If you remove the second value and only include the first value, why does this require anything from the users? Don't you compute `first` yourself?

Reply:
Ying:"The first value is the number of occurrences, the second value is the length of bitvector we assigned. So if the first value is 10, the second value is 4. I was trying to say that, if we only include the first value, when other class want to call the class of ackermannization, it doesn’t know what length we assigned. "

Yoni:"Hmmm. Well… if you save pairs, but the value of the second is a function of the first, then it could be error-prone to save both, because you are counting on the fact that they have this relation, and maybe someplace you forget to update it.
On the other hand, if you really need to access both of them a lot of times, then maybe it makes sense to save both."

Ying:"It is not supposed to be an explicit function, considering for future development.
If we throw one, the information is lost forever. If we really want to keep only one, I would suggest the second one"

Yoni:"My only concern is that a future developer sees that there is a connection between first and second, relies on this in the code, but doesn’t bother to maintain this connection."

@4tXJ7f We want to hear your opinion about this issue. Should we just keep the first number, or both, or just the second?

Reply:
I tend to agree with @yoni206 here. Since the second number is a simple function of the first, it would be better to just define a function that looks up the value in the map and transforms it to a bit-width. If we ever need a more complicated mapping, we can still re-introduce the second value. In general, I would advise not to over-engineer for future development if it adds complexity (this is just a rule of thumb of course and your mileage may vary :) ).

Reply:
"If we throw one, the information is lost forever."
Maybe a way to keep this information is to have it in the documentation of the function that @4tXJ7f suggests to add.